src/maintainer/knowledge_base.rst

```diff
@@ -45,6 +45,7 @@ Some optional, but useful CMake options:
     - ``-DCMAKE_INSTALL_LIBDIR=lib`` Libraries will land in $PREFIX/lib, sometimes projects install
       into lib64 or similar but on conda-forge we keep shared libraries in simply lib.
     - ``-DBUILD_SHARED_LIBS=ON`` Instruct CMake to build shared libraries instead of static ones.
+    - ``${CMAKE_ARGS}`` Add variables defined by conda-forge internally. This is required to enable various conda-forge enhancements, like `CUDA builds <https://conda-forge.org/docs/maintainer/knowledge_base.html#cuda-builds>`__.
```

Comment:
Not completely sure about the wording as I don't fully know what `CMAKE_ARGS` does. Happy about any suggestions.

Reply:
This seems fine. The main thing is we have some text in here. People can always iterate and refine this later